oh_staff_ui/views_utils.py

```diff
@@ -608,3 +611,42 @@ def add_oai_envelope_to_mods(ohmods: OralHistoryMods) -> etree.Element:
     record_el.append(metadata_el)
 
     return record_el
+
+
+def user_in_oh_staff_group(user: User) -> bool:
+    return user.groups.filter(name="Oral History Staff").exists()
+
+
+def delete_file_and_children(media_file: MediaFile) -> None:
+    # if file has child files, delete them first
+    children = list(MediaFile.objects.filter(parent=media_file))
+    for child in children:
+        # first delete file from file system
+        if child.file:
```

Comment:
This check doesn't do what you expect.  Django `FieldFile` objects (which is what `MediaFile.file` contains) only have metadata about the file.  They **should** point to real files... but nothing enforces that beyond the instant the `FieldFile` was created.

For the logging to be correct, you need to build an appropriate `Path` object and check its existence.  But as noted elsewhere, the actual deletion of these should be done via `MediaFile.file.delete()`.

In the Django shell:
```
from pathlib import Path
from django.conf import settings
from oh_staff_ui.models import MediaFile
mf = MediaFile.objects.last()

mf.id, mf.file
# Values will vary, but this is via a very recent local import of production data
# (25005, <FieldFile: oh_static/pdf/submasters/21198-zz002kqdx9-1-submaster.pdf>)

# Django confirms that `mf.file`... is not `None`, but that's all:
bool(mf.file)
# True
bool(mf.file is None)
False
if mf.file: print("exists... or does it?")
# exists... or does it?

# But the file system disagrees
p = Path(settings.MEDIA_ROOT + "/" + mf.file.name)
# PosixPath('secret_media/oh_static/pdf/submasters/21198-zz002kqdx9-1-submaster.pdf')
p.exists()
# False

# My local copy of files does not include any PDFs, including the one
# pointed to by mf.file (wildcard here to avoid possible sequence mismatches)
# $ ll secret_media/oh_static/pdf/submasters/21198-zz002kqdx9*
# ls: cannot access 'secret_media/oh_static/pdf/submasters/21198-zz002kqdx9*': No such file or directory

# File system is correct.  Django data is correct... but locally, not connected to a matching file system.
```